docker/k8s/README.md

```diff
@@ -61,8 +58,16 @@ in SGX-SIM mode.
     [23:20:36 INFO    utility.utility] Decryption Result at Client - You have a risk of 71% to have heart disease. 
     ```
 
-## Heads up
-- The size of LMDB specified as 1TB in 
-  [lmdb_store.cpp](../../tc/sgx/common/packages/db_store/lmdb_store.cpp) should 
-  be decreased to something like accordingly for local development
+## Create Avalon shell pod
+1. Avalon shell pod can be useful to run applications manually.
+    ```bash
+    kubectl create -f shell.yaml
+    ```
+2. After verifying avalon shell pod is running get a shell to the running container
+    ```bash
+    # Let's say 'avalon-shell-abcde' is the pod created above
+    kubectl exec -it avalon-shell-abcde -- /bin/bash
+    ```
+
+# Heads up
```

Comment:
Please add steps to run the generic_client command

Reply:
This is not related to running any specific application. Shell can be used to run any application manually. I've mentioned in in step 1.